packages/@aws-cdk/cx-api/test/fixtures/missing-context/manifest.json

```diff
@@ -16,9 +16,6 @@
     "MyStackName1234": {
       "type": "aws:cloudformation:stack",
       "environment": "aws://37736633/us-region-1",
-      "missing": {
```

Comment:
@eladb @rix0rrr This fails schema validation now since this property is not part of [ArtifactManifest](https://github.com/aws/aws-cdk/blob/master/packages/@aws-cdk/cx-api/lib/cloud-artifact.ts#L31). Was I right to assume this is just leftovers from earlier manifest versions? and passes tests because we had no validation?

Reply:
I believe you are right, but this does raise a point about the fact that the validation might be too strict. I would argue that unknown fields should not cause validation to fail. Is that easy to achieve.

Reply:
👍